packages/jobs/src/adapters/PrismaAdapter/PrismaAdapter.ts

```diff
@@ -224,13 +223,11 @@ export class PrismaAdapter extends BaseAdapter<PrismaAdapterOptions> {
       lockedAt: null,
       lockedBy: null,
       lastError: `${error.message}\n\n${error.stack}`,
-      runAt: null,
+      runAt: new Date(
+        new Date().getTime() + this.backoffMilliseconds(job.attempts),
+      ),
     }
 
-    data.runAt = new Date(
```

Comment:
Was there a reason this was assigned later instead of being part of the initialization?

Reply:
This function used to decide later if `runAt` should be set or left `null`, but now it always sets it, so yep this can be moved up!